src/main/scala/NumberFormat.scala

```diff
@@ -0,0 +1,42 @@
+package CSVValidation
+
+import Errors.NumberFormatError
+import com.ibm.icu.text.DecimalFormat
+
+case class NumberFormat(pattern:Option[String], groupChar: Option[Char] = None, decimalChar: Option[Char] = None) {
+  private var format:String = _
```

Comment:
Is the `format` variable actually used (i.e. read from)? Do we need the `getFormat` method?

Reply:
It is not used, I have removed it now. It was kept there in the idea of making this class similar to DateFormat